Metrics in cm (#3855)
strimzi/strimzi-kafka-operator: documentation/modules/appendix_crds.adoc

|string
|====

[id='type-JmxPrometheusExporterMetrics-{context}']
### `JmxPrometheusExporterMetrics` schema reference

Used in: xref:type-CruiseControlSpec-{context}[`CruiseControlSpec`], xref:type-KafkaClusterSpec-{context}[`KafkaClusterSpec`], xref:type-KafkaConnectS2ISpec-{context}[`KafkaConnectS2ISpec`], xref:type-KafkaConnectSpec-{context}[`KafkaConnectSpec`], xref:type-KafkaMirrorMaker2Spec-{context}[`KafkaMirrorMaker2Spec`], xref:type-KafkaMirrorMakerSpec-{context}[`KafkaMirrorMakerSpec`], xref:type-ZookeeperClusterSpec-{context}[`ZookeeperClusterSpec`]

> The `type` property is a discriminator that distinguishes the use of the type `JmxPrometheusExporterMetrics` from other subtypes which may be added in the future.

Reviewer: ```suggestion
The `type` property is a discriminator that distinguishes use of the `JmxPrometheusExporterMetrics` type from other subtypes which may be added in the future.
```

Author: This is generated for all types :O

> It must have the value `jmxPrometheusExporter` for the type `JmxPrometheusExporterMetrics`.

Reviewer: ```suggestion
The value must be `jmxPrometheusExporter` for the `JmxPrometheusExporterMetrics` type.
```

[options="header"]
|====
|Property          |Description
|type       1.2+<.<|Must be `jmxPrometheusExporter`.
|string
|valueFrom  1.2+<.<|ConfigMap where Prometheus JMX Exporter configuration is stored. See https://github.com/prometheus/jmx_exporter for details of the structure of this configuration.
|xref:type-ExternalConfigurationMetrics-{context}[`ExternalConfigurationMetrics`]
|====

[id='type-ExternalConfigurationMetrics-{context}']
### `ExternalConfigurationMetrics` schema reference

Used in: xref:type-JmxPrometheusExporterMetrics-{context}[`JmxPrometheusExporterMetrics`]

[options="header"]
|====
|Property                |Description

> |configMapKeyRef  1.2+<.<|Reference to the key in the ConfigMap containing the metrics configuration. See external documentation of https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#configmapkeyselector-v1-core[core/v1 configmapkeyselector].

Reviewer: ```suggestion
|configMapKeyRef  1.2+<.<|Reference to the key in the ConfigMap containing the metrics configuration. For more information, see the https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#configmapkeyselector-v1-core[external documentation for core/v1 configmapkeyselector].
```
